qt/place_page_dialog_common.cpp

```diff
@@ -26,6 +28,7 @@ void addCommonButtons(QDialog * this_, QDialogButtonBox * dbb, bool shouldShowEd
 
   QPushButton * routeToButton = new QPushButton("Route To");
   routeToButton->setIcon(QIcon(":/navig64/point-finish.png"));
+  routeToButton->setAutoDefault(false);
```

Comment:
Why these calls are needed? To avoid capturing "Enter"?

Reply:
Yes, if the user doesn't move the cursor, then the enter would activate the first where the autodefault is not disabled.